packages/apputils/src/clientsession.tsx

```diff
@@ -579,13 +577,13 @@ export class ClientSession implements IClientSession {
   /**
    * Select a kernel.
    *
-   * @param cancellable: whether the dialog should have a cancel button.
+   * @param cancelable: whether the dialog should have a cancel button.
```

Comment:
Changed from British spelling to American because that's what we have in the rest of the codebase.

Reply:
Wow, I like the British spelling better. I agree with the reasoning, though.